integration-tests/kyuubi-hive-it/src/test/scala/org/apache/kyuubi/it/hive/operation/KyuubiOperationHiveEnginePerConnectionSuite.scala

```diff
@@ -15,21 +15,25 @@
  * limitations under the License.
  */
 
-package org.apache.kyuubi.operation.hive
+package org.apache.kyuubi.it.hive.operation
 
 import org.apache.commons.lang3.{JavaVersion, SystemUtils}
 
 import org.apache.kyuubi.{Utils, WithKyuubiServer}
 import org.apache.kyuubi.config.KyuubiConf
-import org.apache.kyuubi.config.KyuubiConf.{ENGINE_IDLE_TIMEOUT, ENGINE_SHARE_LEVEL, ENGINE_TYPE}
+import org.apache.kyuubi.config.KyuubiConf._
+import org.apache.kyuubi.it.hive.HiveITMixin
 import org.apache.kyuubi.operation.HiveJDBCTestHelper
 
-class KyuubiOperationHiveEnginePerConnectionSuite extends WithKyuubiServer with HiveJDBCTestHelper {
+class KyuubiOperationHiveEnginePerConnectionSuite extends WithKyuubiServer
+  with HiveJDBCTestHelper with HiveITMixin {
+
   override protected val conf: KyuubiConf = {
     val metastore = Utils.createTempDir(namePrefix = getClass.getSimpleName)
     metastore.toFile.delete()
     val currentUser = Utils.currentUser
     KyuubiConf()
+      .set(s"$KYUUBI_ENGINE_ENV_PREFIX.$KYUUBI_HOME", kyuubiHome)
```

Comment:
what does $KYUUBI_HOME do at engine side

Reply:
Kyuubi server need it to locate the engine jar

Reply:
can you show me the code path?